sir_cake/store/utils.py

```diff
@@ -0,0 +1,79 @@
+from django.contrib import messages
+from django.core.exceptions import ObjectDoesNotExist
+
+from seller.models import Item, OrderPackage
+from users.models import AnonymousUser
+from .models import Basket
+
+
+def add_one_item_basket(request, context):
+    # Anonymous user without session
+    if request.session.session_key is None:
```

Comment:
I suggest creating a function that has a descriptive name and use that in if checks, e.g.:
```python
def anonymous_user_without_session(request):
    return request.session.session_key is None

if anonymous_user_without_session(request):
    ....
```